packages/okta-vue/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@okta/okta-vue",
+  "version": "0.1.0",
```

Comment:
is this the version we want?

Reply:
I figured this is a good starting point. Do you or @robertjd have a preference? I'll defer to you both.

Reply:
That's fine with me. Our other libs use the patch version, but everything is independent now so I think we should do what makes sense for this lib.